storage_image_product/migrations/13.0.1.3.0/post-migration.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# Part of ForgeFlow. See LICENSE file for full copyright and licensing details.
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
```

Comment:
@JordiBForgeFlow `openupgradelib`should be declared as the external dependency into the __manifest__.py file.

Nevertheless, do we really need an other dependecy? We can achieve the same result with the same number of lines of code by using the odoo api...
 

```python

from odoo import SUPERUSER_ID, api

def migrate(cr, version)
    env = api.Environment(cr, SUPERUSER_ID, {}) 

```